grants/faq.md

```diff
@@ -0,0 +1,153 @@
+</details>
+
+<details>
+  <summary><b>I found one of my deliverables to be unnecessary, impossible or already done elsewhere. What do I do?</b></summary>
+
+  Plans change. If you find parts of your original grant application to be unnecessary or you decide to pivot, but you still want to finish the project: get in touch with us. If your new plans are in line with the Web3 Foundation’s values and the council approves the amendment, you can continue your work. If your plans change significantly or you find yourself not being able to finish the grant, we can mutually agree to terminate the grant early. You are always welcome to reapply another time.
+</details>
+
+<details>
+  <summary><b>I am starting a company that [...]. I want to use Polkadot/Kusama/Substrate to build a blockchain/parachain and connect [...]. Would I be eligible for a grant?</b></summary>
+
+  What the Web3 Foundation is mainly looking for to support are projects "[driving advancement and adoption of decentralized software protocols [and] that make it easier for developers to build useful applications using these protocols.](https://web3.foundation/grants/)" As such, we do not give grants to individual companies developing their private infrastructure. However, if part of your work is to build a library or other piece of code that could be of interest to the general Polkadot/Kusama/Substrate ecosystem and ask for funding specific to that, we are happy to look into it. Our [tech stack](https://github.com/w3f/General-Grants-Program/blob/master/grants/polkadot_stack.md) lists some applications and application-oriented grants we have given.
+</details>
+
+<details>
+  <summary><b>Can I list the Web3 Foundation as a partner?</b></summary>
+
+  No. Once the grants team has accepted your first milestone, you may display our [grants badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md) in a project-specific context, such as the repository containing the grant project work.
+</details>
+
+<details>
+  <summary><b>Can you help me advertise my project?</b></summary>
+
+  The Web3 Foundation does not provide PR services to its grantees. However, once per month we will publish all newly signed grants on Twitter. Furthermore, if you would like our opinion on how to write about your grant, we have some general [announcement guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/announcement-guidelines.md). This document also lists an email address through which you can get in touch with our PR team, in case you have any specific questions.
```

Comment:
```suggestion
  The Web3 Foundation does not provide PR services to its grantees. However, once per month we will publish all newly signed grants on Twitter. Furthermore, if you would like our opinion on how to write about your grant, we have some general [announcement guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/announcement-guidelines.md). This document also lists an email address through which you can get in touch with our PR team in case you have specific questions.
```